config/crd/bases/risingwave.risingwavelabs.com_risingwaves.yaml

```diff
@@ -40324,6 +40324,14 @@ spec:
                   even if it's set to true, the controller will determine if it can
                   create the resource by checking if the CRDs are installed.
                 type: boolean
+              enableEmbeddedConnector:
+                default: false
```

Comment:
The default value is false. Does it want to make it compatible with the previous version? Otherwise, it seems true would be a better default value.

Reply:
> Does it want to make it compatible with the previous version?

Yes. It will be deprecated afterwards. See the tracking issue [here](https://github.com/risingwavelabs/risingwave-operator/issues/543).